src/shared/components/toast/toast.styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import { color } from '../../../theme';
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  gap: 15px;
+  background: ${color.mineShaft};
+  padding: 16px 20px;
+  border-radius: 8px;
+  width: 336px;
+  max-width: 100%;
+  min-height: 74px;
+  position: relative;
+  &::before {
+    content: '';
+    display: inline-block;
+    height: 100%;
+    width: 6px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: ${({ color }) => color};
+    border-top-left-radius: 8px;
+    border-bottom-left-radius: 8px;
+  }
+`;
+
+export const CloseButton = styled.button`
+  background-color: transparent;
+  border: none;
+  :hover {
+    cursor: pointer;
+  }
+  svg line {
+    stroke-width: 3;
+  }
+`;
+
+export const IconContainer = styled.div`
+  color: #fff;
```

Comment:
Please use a color object from theme for all of these colors here

Reply:
done ✅